server/database/database.js

```diff
@@ -29,20 +30,26 @@ const ReviewSchema = mongoose.Schema({
 });
 const Review = mongoose.model('Review', ReviewSchema);
 
+
 const MovieSchema = mongoose.Schema({
   id: Number,
+  //need to modify later like reviews
+  suggested: Array,
+  trending: Array,
+  history: Array,
+  //modify --- needs to happen - rob being lazy atm
   mediaType: String,
   title: String,
   rating: Number,
   summary: String,
   reviews: [ReviewSchema],
   imgUrl: String,
-  hulu: Boolean,
-  disneyPlus: Boolean,
-  netflix: Boolean,
-  hboMax: Boolean,
-  appleTvPlus: Boolean,
-  amazonPrimeVideo: Boolean
+  hulu: String,
+  disney: String,
+  netflix: String,
+  hbo: String,
+  apple: String,
+  amazon: String
```

Comment:
Just want to make sure I understand correctly... if there is a string (of an image), the movie is available on that subscription. if there is no string/null, the movie is not available on that subscription?

Reply:
Is the idea to store URLs here?

Reply:
yep, ill be doing the provider search when the search happens. So any providers that movie can be seen on will be stored at that time